xonsh/aliases.py

```diff
@@ -243,13 +264,26 @@ def mpl(args, stdin=None):
         'rmdir',
         'time',
         'type',
-        'vol'
+        'vol',
     }
 
+    if 'Anaconda' in sys.version:
+        def _source_cmd_keep_prompt(args,stdin=None, ):
+            p = builtins.__xonsh_env__.get('PROMPT')
+            source_cmd(args,stdin=stdin)
+            builtins.__xonsh_env__['PROMPT'] = p
+
+        DEFAULT_ALIASES['_source_cmd_keep_prompt'] = _source_cmd_keep_prompt
+
+        DEFAULT_ALIASES['activate'] = ['_source_cmd_keep_prompt', 'activate.bat']
+        DEFAULT_ALIASES['deactivate'] = ['_source_cmd_keep_prompt', 'deactivate.bat']
+
+
     for alias in WINDOWS_CMD_ALIASES:
         DEFAULT_ALIASES[alias] = ['cmd', '/c', alias]
 
     DEFAULT_ALIASES['which'] = ['where']
+    DEFAULT_ALIASES ['source_cmd'] = source_cmd
```

Comment:
This should be `source-cmd`, rather than `source_cmd` to be consistent with the other source aliases.